src/System.Memory/src/System/ReadOnlySpan.cs

```diff
@@ -358,7 +358,10 @@ public T[] ToArray()
         /// <summary>
         /// Returns a 0-length read-only span whose base is the null pointer.
         /// </summary>
-        public static readonly ReadOnlySpan<T> Empty = default(ReadOnlySpan<T>);
+        public static ReadOnlySpan<T> Empty
```

Comment:
Nit: The rest of this file is written using the new C# syntax for simple property getters, so it would be nice to follow it here as well:

```
public static ReadOnlySpan<T> Empty => default(ReadOnlySpan<T>);
```